cmd/bosun/web/static/partials/incident.html

```diff
@@ -0,0 +1,24 @@
+<div class="row" ng-show="error">
+	<div class="col-lg-12">
+		<div class="alert alert-danger" ng-bind="error"></div>
+	</div>
+</div>
+<div ng-show="incident">
+	<div class="row">
+		<h3> Incident {{incident.Id}}: {{incident.AlertKey}} </h3>
+	</div>
+	<div class="row">
+		<h4 ng-show="incident.End"><span ts-time="incident.Start" ts-end-time="incident.End" no-link="true"></span></h4>
+		<h4 ng-hide="incident.End"><span ts-time="incident.Start" no-link="true" /> (Open)</h4>
```

Comment:
Why `no-link`? Linking to the event start time is useful. Or did I forget what that directive does?